### PR TITLE
Styles api

### PR DESCRIPTION
**Pull request by dazyang**
**KEY CHANGES**
- Added Social Icons from FontAwesome

- Refracted ProfileCard component by recomposing profile description and profile header into two seperate molecule components. 

- Added back Ambassador components to the file

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -28,44 +28,90 @@
...
+.ambassadors {
+  display: flex;
+  flex-flow: row wrap;
+  /* width: 85%; */
+  height: 100px;
+  -webkit-margin-before: 1em;
+  -webkit-margin-after: 1em;
+  -webkit-margin-start: 0px;
+  -webkit-margin-end: 0px;
+  -webkit-padding-start: 40px
```
Comment (agentlewis) [inline src/App.css]: you shouldn't need to do prefixing as create react app has this already built in. see https://github.com/facebook/create-react-app#whats-included

Also currently the way this code is writ non-webkit browsers wont have margin or padding as you have described - is that intentional? if its dealing with a specific problem you can leave a comment here describing why.

File: src/App.css [hunk 2]
```diff
@@ -28,44 +28,90 @@
   to { transform: rotate(360deg); }
 }
 
+.ambassadors {
+  display: flex;
+  flex-flow: row wrap;
+  /* width: 85%; */
```
Comment (agentlewis) [inline src/App.css]: We should remove this commented out code :)

File: src/App.css [hunk 3]
```diff
@@ -28,44 +28,90 @@
...
+}
+
 .profile-cards {
   margin: 10px;
-  padding: 10px;
-  width: 200px;
+  width: 220px;
   background: rgb(220, 225, 223);
-  box-shadow:  0.5px 0.5px 0.5px 0.5px grey;
+  /* box-shadow:  0.5px 0.5px 0.5px 0.5px grey; */
```
Comment (agentlewis) [inline src/App.css]: remove :)

File: src/App.css [hunk 4]
```diff
@@ -28,44 +28,90 @@
...
-  height: 20px;
-  margin: 1px;
-}
 
 .profile-description {
   margin-bottom: 10px;
+  padding: 10px 8px 8px 8px;
+}
+
+#description {
```
Comment (agentlewis) [inline src/App.css]: Is there any reason description is an id and not a class. In general the only times to use id's is when the item should be find able with a #link. eg https://github.com/facebook/create-react-app#whats-included

File: src/App.css (hunk 1 above)
Comment (dazyang) [reply src/App.css]: I forgot to go back to review this bit. I wanted to match Ambassador with Masonry's styling on members and contributors block. So I hunted those down in DOM and copied and pasted them.

File: src/App.css (hunk 4 above)
Comment (dazyang) [reply src/App.css]: Forgot why I did that. I have removed now and moved the styling into `profile-description`. :)

**Pull request merged**